build/distributions/CentOS/6/x86_64/yum_data/packages.yaml

```diff
@@ -151,46 +151,46 @@ pdsh-rcmd-ssh:
-  :source: http://mirror.cogentco.com/pub/linux/epel/6/x86_64/perl-Net-FTP-RetrHandle-0.2-3.el6.noarch.rpm
+  :source: http://lug.mtu.edu/epel/6/x86_64/Packages/p/perl-Net-FTP-RetrHandle-0.2-3.el6.noarch.rpm
 perl-Net-SNMP:
   :rpm_name: perl-Net-SNMP-5.2.0-4.el6.noarch.rpm
-  :source: http://mirror.cogentco.com/pub/linux/epel/6/x86_64/perl-Net-SNMP-5.2.0-4.el6.noarch.rpm
+  :source: http://lug.mtu.edu/epel/6/x86_64/Packages/p/perl-Net-SNMP-5.2.0-4.el6.noarch.rpm
 perl-Sort-Versions:
   :rpm_name: perl-Sort-Versions-1.5-12.el6.noarch.rpm
-  :source: http://mirror.cogentco.com/pub/linux/epel/6/x86_64/perl-Sort-Versions-1.5-12.el6.noarch.rpm
+  :source: http://lug.mtu.edu/epel/6/x86_64/Packages/p/perl-Sort-Versions-1.5-12.el6.noarch.rpm
 perl-Time-modules:
   :rpm_name: perl-Time-modules-2006.0814-5.el6.noarch.rpm
   :source: http://mirror.ash.fastserv.com/pub/linux/centos/6.9/os/x86_64/Packages/perl-Time-modules-2006.0814-5.el6.noarch.rpm
 postgresql96:
-  :rpm_name: postgresql96-9.6.4-1PGDG.rhel6.x86_64.rpm
-  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6.9-x86_64/postgresql96-9.6.4-1PGDG.rhel6.x86_64.rpm
+  :rpm_name: postgresql96-9.6.9-1PGDG.rhel6.x86_64.rpm
+  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6.9-x86_64/postgresql96-9.6.9-1PGDG.rhel6.x86_64.rpm
 postgresql96-contrib:
-  :rpm_name: postgresql96-contrib-9.6.4-1PGDG.rhel6.x86_64.rpm
-  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6.9-x86_64/postgresql96-contrib-9.6.4-1PGDG.rhel6.x86_64.rpm
+  :rpm_name: postgresql96-contrib-9.6.9-1PGDG.rhel6.x86_64.rpm
+  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6.9-x86_64/postgresql96-contrib-9.6.9-1PGDG.rhel6.x86_64.rpm
 postgresql96-libs:
-  :rpm_name: postgresql96-libs-9.6.4-1PGDG.rhel6.x86_64.rpm
-  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6.9-x86_64/postgresql96-libs-9.6.4-1PGDG.rhel6.x86_64.rpm
+  :rpm_name: postgresql96-libs-9.6.9-1PGDG.rhel6.x86_64.rpm
+  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6.9-x86_64/postgresql96-libs-9.6.9-1PGDG.rhel6.x86_64.rpm
 postgresql96-server:
-  :rpm_name: postgresql96-server-9.6.4-1PGDG.rhel6.x86_64.rpm
-  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6.9-x86_64/postgresql96-server-9.6.4-1PGDG.rhel6.x86_64.rpm
+  :rpm_name: postgresql96-server-9.6.9-1PGDG.rhel6.x86_64.rpm
+  :source: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6.9-x86_64/postgresql96-server-9.6.9-1PGDG.rhel6.x86_64.rpm
 pssh:
   :rpm_name: pssh-2.3.1-5.el6.noarch.rpm
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/6/pssh-2.3.1-5.el6.noarch.rpm
 puppet-agent:
-  :rpm_name: puppet-agent-1.10.9-1.el6.x86_64.rpm
-  :source: http://yum.puppetlabs.com/el/6/PC1/x86_64/puppet-agent-1.10.9-1.el6.x86_64.rpm
+  :rpm_name: puppet-agent-1.10.10-1.el6.x86_64.rpm
```

Comment:
This is not the latest available (1.10.14-1).  How do we decide which version to use?

Reply:
I was updating to latest, must've missed this one. All puppet-agent packages are now at 1.10.14-1